### ISSUE TITLE
[Critical] Information or relationship only presented visually

### ISSUE THREAD

**Issue by ntallman** [APTrust/dart#614]
## Issue Details

| | |
|---|---|
| **Source Issue ID** | 87760 |
| **Checkpoint** | 1.3 Adaptable › Semantics (1.3.1.a) |
| **Impact** | Critical |
| **Standards** | wcag2a, wcag2aa, wcag21a, wcag21aa, wcag22a, wcag22aa |
| **Test Area** | 03 New Job Workflow |
| **URL** | <https://aptrust.org/documentation-page/dart-digital-archivists-resource-tool/> |
| **Environment** | Production |
| **Platform** | Windows | Chrome | Firefox |
| **Assistive Technology** | JAWS | NVDA | Magnifier |
| **Method** | Manual |
| **Digital Asset Type** | Desktop Web |
| **Release** | 4.2026 |
| **Date Reported** | 05/12/26 at 2:29 pm |
| **Issue Status** | Open |
| **Unit Type** | Page |
| **Issue Type** | Accessibility |
| **Reported By** | kim.casey@ablr360.com |

## Description

Information, structure, or relationship is presented visually but is not conveyed programmatically or in text.
On the Files screen, when you tab into the list of folders, only the folder name is announced.  In this case, it was "/" or backslash, click link to open.  This context is presented visually (off to the right and in the inaccessible tooltip), but this information would be in reading order after the file folder listing. See Create Job User Experience video at 4:45.

## Recommendation

Rule : 
Information, structure, or relationship that is presented visually MUST be conveyed programmatically or in text.

How To Fix: 
Suggest creating a region area for the file selection area.  Add role="region" and aria-label="Select files to add to your package, file navigation"

Fix this issue by doing ONE OR MORE of the following:
1. Use a text-based method to convey the meaning of the visual information or relationship. The text can be provided for screen readers only or be visible on screen.
2. When available in HTML, use appropriate semantic HTML elements.
3. Use ARIA to provide programmatic information for screen readers. (Caution: Be certain you know which ARIA attributes can be used on which HTML elements and test the results with screen readers.)

Reference: 
ARIA: region role - https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Reference/Roles/region_role

Background: 
People who are blind cannot rely on the visual appearance and layout of a page to understand information, relationships, or structure that is conveyed only visually. In order to convey visual information to screen reader users it must be also provided programmatically or in text. This will allow screen reader users to understand information, relationships, or structure that is apparent visually.

## Screenshots

![03_relationship visual](https://raw.githubusercontent.com/APTrust/dart/master/docs/accessibility-audit/screenshots/87760/03_relationship_visual.png)
